Codigo/SistemaCombate.bas

```diff
@@ -1833,6 +1833,45 @@ Public Function PuedeAtacarNPC(ByVal AttackerIndex As Integer, ByVal NpcIndex As
                         Exit Function
                 End Select
             End If
+            
+            
+            If NpcList(NpcIndex).flags.AttackedBy <> "" Then
+                Dim AttackedBy As t_UserReference
+                Dim AttackedByIndex As Integer
+                Dim Lider As t_User
+                Dim a As Integer
+                
+
+                    AttackedBy = NameIndex(NpcList(NpcIndex).flags.AttackedBy)
+                    If IsValidUserRef(AttackedBy) Then
```

Comment:
this is confusing, can we replace attackedby wih CurrentOwner?

Reply:
ok